WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1682,7 +1684,7 @@ ChooserDialogBody                    = width:320px; @Text@
 ChooserDialogInputContainer          = padding-bottom:3px; padding-top:3px;
 ChooserDialogInputField              = @borderRadius@:2px; @PageBg@; @SeparatorBorder@; width:320px; height:32px; margin-bottom:16px;
 ChooserDialogItemContainer           = background-color:@AppC@; overflow:auto;
-ChooserDialogItemInnerContainer      = @SeparatorBorder@; padding:18px 0px 0px 16px; border-color:@SeparatorColor@; @borderRadius@:2px;
+ChooserDialogItemInnerContainer      = @SeparatorBorder@; border-color:@SeparatorColor@; @borderRadius@:2px;
```

Comment:
do we need border-color here? @SeparatorBorder@ internally uses @SeparatorColor@

Reply:
Removed